cmd/f3/msgdump/envelope.go

```diff
@@ -1,75 +0,0 @@
-package msgdump
```

Comment:
Rewrote the stuff here, because:
* there seem to be an [issue with decoding parquet files previously generated](https://github.com/filecoin-project/go-f3/pull/731).  So far we have just been using the ndjson dump to query things.
* I wanted to have a SQL interface that: 1) can query the latest messages and 2) can query the rotated messages all in one convenient view.
* These changes are significant enough that it deserved its own package etc.